test/acceptance/tests/basic/main_test.go

```diff
@@ -0,0 +1,15 @@
+package basic
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/consul-helm/test/acceptance/framework"
+)
+
+var suite framework.Suite
```

Comment:
yes, I totally understand. Maybe this seems weird because the `MainTest` and the "actual" tests are in different files. Would you feel differently if they all were in one file? We're using global `var`s all the time in consul-k8s tests ([example](https://github.com/hashicorp/consul-k8s/blob/9bbf6df4e23b9b1a6bf251fe88215328b967520d/subcommand/server-acl-init/command_test.go)). 

Also, as an example, you could look at istio's integration tests, where they have global vars in the `main_test.go` file; those vars are shared and used by the tests within a package. [Here](https://github.com/istio/istio/blob/8f24e6ca078ac12c02f31e5bc00b038a1359d472/tests/integration/pilot/main_test.go) is an example.

The parallelism concerns would be similar to having global vars in a single test file: you'd have to make sure no two tests are writing to it at the same time. In our case, we'd have to be careful that tests aren't modifying the environment and are just reading info they need. I could be missing something here, but I think we can make it work.

Reply:
Gotcha, so as long as we don't have any mutation of `suite` then we're good.

Reply:
yes, I believe so.